src/main/java/org/sagebionetworks/bridge/models/schedules2/adherence/weekly/WeeklyAdherenceReport.java

```diff
@@ -46,6 +49,8 @@
     private String userId;
     @Embedded
     private AccountRef participant;
+    @Enumerated(EnumType.STRING)
+    private ParticipantProgressionState progression;
```

Comment:
This field could be added to WeeklyAdherenceReportTest.

Reply:
Added.